hrm-service/src/routes/v0/contacts.js

```diff
@@ -25,14 +25,19 @@ contactsRouter.put('/:contactId/connectToCase', publicEndpoint, async (req, res)
   const { accountSid, user } = req;
   const { contactId } = req.params;
   const { caseId } = req.body;
-  await CaseController.getCase(caseId, accountSid);
-  const updatedContact = await ContactController.connectToCase(
-    contactId,
-    caseId,
-    accountSid,
-    user.workerSid,
-  );
-  res.json(updatedContact);
+  try {
+    const updatedContact = await ContactController.connectToCase(
+      contactId,
+      caseId,
+      accountSid,
+      user.workerSid,
+    );
+    res.json(updatedContact);
+  } catch (err) {
+    if (err.message.includes('violates foreign key constraint')) {
```

Comment:
The foreign key constraint will reject any attempts to connect a contact to a non-existent case, so the first query seemed redundant?